docs/admin_guide.md

```diff
@@ -85,6 +85,12 @@ The currently supported types for AID columns are: `integer`, `bigint`, `text` a
 
 The procedure `diffix.unmark_table(table_name)` clears the labels for the table and all its AID columns.
 
+Non-AID columns from personal tables can be marked as `not_filterable` in order to prevent untrusted users from using data filtering conditions over them in their anonymizing queries.
+
+This can be accomplished by calling the procedure `diffix.mark_not_filterable('table_name', 'column_name')`. The procedure `diffix.mark_filterable` clears the previously set label for a column.
+
+Columns in which a single value dominates the rest are good candidates for being restricted in such a way.
```

Comment:
Maybe change the last line to this:

There are rare but possible cases where, under certain data conditions and analyst knowledge, an inference about a single user may be possible. The data conditions occur when a column is dominated by a single value (e.g. one value constitutes greater than 50% of the rows). Columns with this condition are good candidates for this restriction.

Reply:
I can't change a PR in the Web UI, so I will merge this as is and will create another one for the updated text.